Packs/Endace/Playbooks/license.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2020 Endace Technology Limited

```

Comment:
not sure why this file is needed by please rename it to have the playbook filename as prefix with `_` as separator